runtime/jniinv/main.c

```diff
@@ -31,8 +31,6 @@
 #include "j9cfg.h"
 #include "errno.h"
 
-
-#define JNI_VERSION_1_3 0x00010003
```

Comment:
There is some code below in comments that uses this, that should be deleted as well.

Reply:
Or the define moved into the commented code, since the test is still valid.

Reply:
I thought the declaration was just redundant: I removed the use in the comment as well.

Reply:
I think the test case should be restored and the constant added into the commented out code.

Reply:
`JNI_VERSION_1_3` is not defined in `jni.h` for any version OpenJ9 supports.

Reply:
and the test case confirms that JNI_VERSION_1_3 is rejected.

Reply:
Ok, I'll just revert that file.